src/components/Releases/index.tsx

```diff
@@ -14,23 +26,19 @@ type ReleaseData = {
     signature: string,
 }
 
-function createReleaseData(index: number, version: string): ReleaseData {
-    var vtag = `${version}-incubating`
-    // Drop the incubating suffix for releases after graduation
-    if (index >= 4) {
-        vtag = `${version}`
-    }
-    const archive = `https://downloads.apache.org/kvrocks/${version}/apache-kvrocks-${vtag}-src.tar.gz`
+function createReleaseData(version: string, vtag?: string): ReleaseData {
+    const fixedVTag = vtag ?? version;
```

Comment:
I noticed that the downloads link are use `2.4.0` so we do have two concepts here. But later we can populate only the `version` field and if we add a `semver` dependency, we can do a test against version string (anyway, not depend on array index).